src/targets/gpu/mlir.cpp

```diff
@@ -1096,6 +1097,46 @@ std::string dump_mlir(module m, const std::vector<shape>& inputs)
     return mlir_print(&mlirOperationPrint, mod_op);
 }
 
+static std::string compute_dump_name(const module& m, const std::string& ext)
+{
+    std::vector<instruction_ref> sizes;
+    for(auto ins : iterator_for(m))
+    {
+        if(not contains({"convolution", "dot"}, ins->name()))
```

Comment:
`quant_convolution` and `quant_dot` need to be added to the list.

Reply:
Why not just search for the term `mlir`, if that is available at this stage?

Reply:
(I think that would be either/or situation). If `mlir` can be searched for, that should be enough.. @pfultz2 ?

Reply:
There is no mlir op.

> Should I add quant_convolution and quant_dot to the .mxr (not mlir) file dumping function?

This is just for adding the sizes, which should be the same for both mxr and mlir file.